src/store/actions/comments.actions.js

```diff
@@ -33,6 +33,19 @@ export default {
     })
   },
 
+  removeComment: ({commit, getters, state}, { id }) => {
+    console.log('ID', id)
+    let token = getters.getToken
+    if (!token) {
+      return
```

Comment:
I was thinking we could throw an error here instead? That way we know to handle that in the view (eventually we want to do the store { glboalError: 'This error stays but can be dimissed', pageError: 'some page error that goes away after navigating'}